sim/team-validator.js

```diff
@@ -841,6 +841,15 @@ class Validator {
 		do {
 			alreadyChecked[template.speciesid] = true;
 			if (dex.gen === 2 && template.gen === 1) tradebackEligible = true;
+			// STABmons hack to avoid copying all of validateSet to formats
+			if (format.banlistTable && format.banlistTable['ignorestabmoves'] && !(moveid in {'acupressure':1, 'bellydrum':1, 'chatter':1, 'geomancy':1, 'shellsmash':1, 'shiftgear':1, 'thousandarrows':1}) && !move.isZ) {
```

Comment:
maybe define `!(moveid in {'acupressure':1, 'bellydrum':1, 'chatter':1, 'geomancy':1, 'shellsmash':1, 'shiftgear':1, 'thousandarrows':1})` independently to prevent the `if` statement from being crazily long

Reply:
I just copied and pasted that from when it last was on the server...

Reply:
yeah i can tell but i also know that more moves will be banned in the future, so i was just suggesting something

Reply:
We could do what I did with Sketchmons and have it in formats.js, that way you only have to edit one file and random people (EV) won’t have a problem PRing changes.

Reply:
lol